### PR TITLE
update exporting section of step by step for 3.1

### PR DESCRIPTION
**Pull request by skyace65**
Updates part of the exporting section in step by step for Godot 3.1. One image was updated, one was added (both have gone through png compression), and 2 sections were rewritten to reflect changes in 3.1. Closes Issue #2275

### CODE REVIEW

File: getting_started/step_by_step/exporting.rst [hunk 1]
```diff
@@ -28,16 +28,19 @@ or tablet, you need to support touchscreen input. Because a click event can
...
-The first step is to open "Project Settings" and find the *Handheld* section.
-Enable the *Emulate Touchscreen* option. This lets you treat mouse click
-events the same as touch events, so you can test the game on a computer without
-a touchscreen. Also, make sure to select "portrait" under *Orientation*.
-
-In the *Stretch* section, set *Mode* to "2d" and *Aspect* to "keep". This
-ensures that the game scales consistently on different sized screens.
+By default Godot emulates mouse input from touch input. That means if anything
+is coded to happen on a mouse event, touch will trigger it as well. If you wish to
+disable that for whatever reason, or emulate touch from mouse imput, you can do that
```
Comment (cbscribe) [inline getting_started/step_by_step/exporting.rst]: imput -> input

File: getting_started/step_by_step/exporting.rst [hunk 2]
```diff
@@ -28,16 +28,19 @@ or tablet, you need to support touchscreen input. Because a click event can
...
+in the "project settings" under *Input Devices* and *Pointing*
```
Comment (cbscribe) [inline getting_started/step_by_step/exporting.rst]: "Project Settings" should be capitalized.

File: getting_started/step_by_step/exporting.rst [hunk 3]
```diff
@@ -28,16 +28,19 @@ or tablet, you need to support touchscreen input. Because a click event can
...
 
 .. image:: img/export_touchsettings.png
 
+Before we change the input method, open the project settings and go to *Display*,
```
Comment (cbscribe) [inline getting_started/step_by_step/exporting.rst]: Don't we already have project settings open from the previous sentence?

**Comment by skyace65** [comment]
Should be fixed now.

**Comment by cbscribe** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.

**Pull request merged**